lib/internal/Magento/Framework/View/Model/Layout/Merge.php

```diff
@@ -41,9 +43,15 @@ class Merge implements \Magento\Framework\View\Layout\ProcessorInterface
 
     /**
      * Cache id suffix for page layout
+     * @deprecated it was replaced by PAGE_LAYOUT_MERGED_CACHE_SUFFIX
      */
     const PAGE_LAYOUT_CACHE_SUFFIX = 'page_layout';
 
+    /**
+     * Cache id suffix for layout xml and page layout
+     */
+    const PAGE_LAYOUT_MERGED_CACHE_SUFFIX = 'page_layout_merged';
```

Comment:
Hi @andrey-legayev, thanks for collaboration. Can you please explain the reason why you introduce new constant instead of change value of origin constant?

Reply:
AFAIK change of value of existing constant is not backward compatible according to Magneto2 rules

Reply:
@andrey-legayev you're not supposed to mark conversation is resolved by yourself :)

Reply:
sorry for that

Reply:
@andrey-legayev  introducing new constant also is backward incompatible, so in this case, where we can't do it without BC I think the better way is change value of origin constant. Looks like this change shouldn't break customers

Reply:
It's not clear to me how _new_ constant can break backward compatibility.
Can you send me link to text of this rule?

Reply:
I've found that changing const value is Ok.
FYI - there is nothing about adding new constants.
Link: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#changing-the-value-of-a-constant

Reply:
I've removed new constant.

Reply:
Can this conversation be resolved now?